src/scene/materials/standard-material.js

```diff
@@ -498,70 +498,10 @@ let _params = new Set();
- * - diffuseTint: defines if {@link StandardMaterial#diffuse} constant should affect diffuse color.
- * - specularTint: defines if {@link StandardMaterial#specular} constant should affect specular
- * color.
- * - metalnessTint: defines if {@link StandardMaterial#metalness} constant should affect metalness
- * value.
- * - glossTint: defines if {@link StandardMaterial#shininess} constant should affect glossiness
- * value.
- * - emissiveTint: defines if {@link StandardMaterial#emissive} constant should affect emission
- * value.
- * - opacityTint: defines if {@link StandardMaterial#opacity} constant should affect opacity value.
- * - occludeSpecularFloat: defines if {@link StandardMaterial#occludeSpecularIntensity} constant
- * should affect specular occlusion.
- * - alphaTest: enable alpha testing. See {@link Material#alphaTest}.
- * - alphaToCoverage: enable alpha to coverage. See {@link Material#alphaToCoverage}.
- * - opacityFadesSpecular: enable specular fade. See {@link Material#opacityFadesSpecular}.
- * - alphaFade: fade value. See {@link Material#alphaFade}.
- * - sphereMap: if {@link StandardMaterial#sphereMap} is used.
- * - cubeMap: if {@link StandardMaterial#cubeMap} is used.
- * - ambientSH: if ambient spherical harmonics are used. Ambient SH replace prefiltered cubemap
- * ambient on certain platform (mostly Android) for performance reasons.
- * - useSpecular: if any specular or reflections are needed at all.
- * - fixSeams: if cubemaps require seam fixing (see {@link Texture#options.fixCubemapSeams}).
- * - emissiveEncoding: how emissiveMap is encoded. This value is based on Texture#encoding.
- * - lightMapEncoding: how lightMap is encoded. This value is based on on Texture#encoding.
- * - packedNormal: if normal map contains X in RGB, Y in Alpha, and Z must be reconstructed.
- * - forceFragmentPrecision: Override fragment shader numeric precision. Can be "lowp", "mediump",
- * "highp" or null to use default.
- * - fastTbn: Use slightly cheaper normal mapping code (skip tangent space normalization). Can look
- * buggy sometimes.
- * - refraction: if refraction is used.
- * - skyboxIntensity: if reflected skybox intensity should be modulated.
- * - useCubeMapRotation: if cube map rotation is enabled.
- * - useInstancing: if hardware instancing compatible shader should be generated. Transform is read
- * from per-instance {@link VertexBuffer} instead of shader's uniforms.
- * - useMorphPosition: if morphing code should be generated to morph positions.
- * - useMorphNormal: if morphing code should be generated to morph normals.
- * - reflectionSource: one of "envAtlasHQ", "envAtlas", "cubeMap", "sphereMap"
- * - reflectionEncoding: one of null, "rgbm", "rgbe", "linear", "srgb"
- * - ambientSource: one of "ambientSH", "envAtlas", "constant"
- * - ambientEncoding: one of null, "rgbm", "rgbe", "linear", "srgb"
+ * assigned to the material, a reflection pass with simpler shaders and so on. These properties are
```

Comment:
on top of this class is also this code

<img width="741" alt="Screenshot 2022-12-07 at 17 45 26" src="https://user-images.githubusercontent.com/59932779/206255311-8bc90338-a156-4e37-8002-7034120f4e4c.png">

we should replace both `*` by the options type.

Reply:
This still needs to be done too.

Reply:
Yep, fixed.